### PR TITLE
fix(colors): fix selection text color in safari

### PR DESCRIPTION
**Pull request by AlexZakablukov**
**What this PR does** 📖
fix selection text color in safari

**Which issue(s) this PR fixes** 🔨
[AP-1207](https://satellite-im.atlassian.net/browse/AP-1207)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by phillsatellite — APPROVED** [review]

File: layouts/files.vue [hunk 1]
```diff
@@ -86,8 +88,14 @@ export default Vue.extend({
     }
   },
   computed: {
-    ...mapState(['friends']),
+    ...mapState(['friends', 'ui']),
```
Comment (molimauro) [inline layouts/files.vue:91]: lets not spread the whole `ui` here, use directly the store value :)

File: layouts/friends.vue [hunk 2]
```diff
@@ -91,8 +93,14 @@ export default Vue.extend({
     }
   },
   computed: {
-    ...mapState(['friends', 'groups']),
+    ...mapState(['friends', 'groups', 'ui']),
```
Comment (molimauro) [inline layouts/friends.vue:96]: same

File: assets/styles/base.less [hunk 3]
```diff
@@ -37,6 +38,16 @@ html {
   &:extend(.background-flair);
 }
 
+/* Only for Safari to fix selection text background */
+
+@media not all and (min-resolution: 0.001dpcm) {
+  @supports (-webkit-appearance: none) {
+    ::selection {
+      background: rgba(var(--flair-color-rgb), 0.99) !important;
```
[the remark this answers is not in the document]
Comment (AlexZakablukov) [reply assets/styles/base.less:46]: Safari has a special approach to painting text ::selection, it is described [here](https://github.com/w3c/csswg-drafts/issues/6853), when you using fully opaque background color safari add to it his own default opacity, so I made background 99% opacity and this problem is gone.
Also I answered on this question on [stackoverflow](https://stackoverflow.com/questions/71567352/how-do-you-fix-the-darkened-text-selection-color-in-safari-css/71753552?noredirect=1#comment126832563_71753552)

File: layouts/files.vue (hunk 1 above)
Comment (AlexZakablukov) [reply layouts/files.vue]: fixed

File: layouts/friends.vue (hunk 2 above)
Comment (AlexZakablukov) [reply layouts/friends.vue]: fixed

File: assets/styles/base.less (hunk 3 above)
[the remark this answers is not in the document]
Comment (molimauro) [reply assets/styles/base.less:46]: oh okay thanks!

**Review by molimauro — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**